nfts.proto

```diff
@@ -197,5 +197,6 @@ message NftEvents {
     MetaplexMasterEditionTransaction solana_retry_create_collection = 34;
     MintMetaplexMetadataTransaction solana_mint_to_collection = 35;
     MintMetaplexMetadataTransaction solana_retry_mint_to_collection = 36;
+    string import_solana_collection = 37;
```

Comment:
What is this event for?

`finished_importing_solana_collection = 37`

Is this to signal the end of the import?

Reply:
This is emitted when mutation is called. nfts-solana consumes it and starts making api calls for import